{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Under investigation",
      "package_name": "bind",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 9",
      "fix_state": "Under investigation",
      "package_name": "bind",
      "cpe": "cpe:/o:redhat:enterprise_linux:9"
    },
    {
      "product_name": "Red Hat OpenStack Platform 16.1",
      "fix_state": "Under investigation",
      "package_name": "bind",
      "cpe": "cpe:/a:redhat:openstack:16.1"
    },
    {
      "product_name": "Red Hat OpenStack Platform 16.2",
      "fix_state": "Under investigation",
      "package_name": "bind",
      "cpe": "cpe:/a:redhat:openstack:16.2"
    },
    {
      "product_name": "Red Hat OpenStack Platform 17.0",
      "fix_state": "Under investigation",
      "package_name": "bind",
      "cpe": "cpe:/a:redhat:openstack:17.0"
    },
    {
      "product_name": "Red Hat OpenStack Platform 17.1",
      "fix_state": "Under investigation",
      "package_name": "bind",
      "cpe": "cpe:/a:redhat:openstack:17.1"
    },
    {
      "product_name": "Red Hat OpenStack Platform 18.0",
      "fix_state": "Under investigation",
      "package_name": "bind",
      "cpe": "cpe:/a:redhat:openstack:18.0"
    }
  ],
  "threat_severity": "Important",
  "public_date": "2023-06-21T00:00:00Z",
  "bugzilla": {
    "description": "stack overflow",
    "id": "2216576",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2216576"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "7.5",
    "cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
    "status": "draft"
  },
  "iava": "",
  "cwe": "",
  "statement": "",
  "acknowledgement": "",
  "name": "CVE-2023-2911",
  "document_distribution": "",
  "details": [
    "If the `recursive-clients` quota is reached on a BIND 9 resolver configured with both `stale-answer-enable yes;` and `stale-answer-client-timeout 0;`, a sequence of serve-stale-related lookups could cause `named` to loop and terminate unexpectedly due to a stack overflow. This issue affects BIND 9 versions 9.16.33 through 9.16.41, 9.18.7 through 9.18.15, 9.16.33-S1 through 9.16.41-S1, and 9.18.11-S1 through 9.18.15-S1."
  ],
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2023-2911\nhttps://nvd.nist.gov/vuln/detail/CVE-2023-2911"
  ]
}